x-pack/plugins/security_solution/server/endpoint/services/actions/clients/get_response_actions_client.test.ts

```diff
@@ -30,10 +29,4 @@ describe('getResponseActionsClient()', () => {
       );
     }
   );
-
-  it(`should throw error if agentType is not supported`, () => {
```

Comment:
Why was this test removed?

Reply:
The test is redundant as we removed `UnsupportedResponseActionsAgentTypeError` as it was unreachable code.

Reply:
a315f7cbebf883ce3ace397c7f09f3821d761932